src/linker/Linker/Annotations.cs

```diff
@@ -583,10 +582,6 @@ public bool SetPreservedStaticCtor (TypeDefinition type)
 		public bool TryGetLinkerAttribute<T> (IMemberDefinition member, [NotNullWhen (returnValue: true)] out T? attribute) where T : Attribute
 		{
 			var attributes = GetLinkerAttributes<T> (member);
-			if (attributes.Count () > 1) {
```

Comment:
We should probably at least assert that there's only one such attribute. For example DynamicaDependency can still occur multiple times, but this method doesn't handle it correctly - basically it should never be called for attributes which allow multiple instances per member.